conf/aws/application.conf

```diff
@@ -0,0 +1,162 @@
+# 2. Version below is available on the S3 at: https://s3-us-west-2.amazonaws.com/gobblin-libs/latest-jars/
+gobblin.aws.version="0.6.2-701-g7c07fd5"
+
+## AWS cluster configuration
+# NFS parent cannot be changed until logic for determining appWorkDir in gobblin-cluster is not made generic
+gobblin.aws.nfs.parent.dir="/home/ec2-user"
+gobblin.aws.nfs.root.dir=${gobblin.aws.nfs.parent.dir}"/"${gobblin.aws.cluster.name}
+gobblin.cluster.helix.cluster.name=${gobblin.aws.cluster.name}
+gobblin.cluster.job.conf.path=${gobblin.aws.nfs.root.dir}"/job-conf"
+gobblin.cluster.job.configuration.manager="gobblin.aws.AWSJobConfigurationManager"
+gobblin.aws.email.notification.on.shutdown=false
+
+## AWS Configuration
+gobblin.aws.region=us-west-2
+gobblin.aws.client.assume.role=false
+gobblin.aws.client.role.arn="<Role ARN if client assume role is true>"
+gobblin.aws.client.external.id="<External ID if client assume role is true>"
+gobblin.aws.client.session.id="<Session ID if client assume role is true>"
+# Units: ms / millis, us / microseconds, ns / nanoseconds, d / days, h / hours, s / seconds, m / minutes
+# If no unit is specified, it defaults to ms
+gobblin.aws.credentials.refresh.interval=3d
+gobblin.aws.job.conf.refresh.interval=1m
+
+## AWS Master config
+gobblin.aws.master.ami.id=ami-f303fb93
+gobblin.aws.master.instance.type="m3.medium"
+gobblin.aws.master.jvm.args="-XX:ReservedCodeCacheSize=100M"
+gobblin.aws.master.jvm.memory=3G
+# Master jars location is tied closely to AMI being used. Only change if you know what you are doing
+gobblin.aws.master.jars.dir="/home/ec2-user/gobblin-lib"
+
+## AWS Worker config
+gobblin.aws.worker.ami.id=ami-f303fb93
+gobblin.aws.worker.instance.type="m3.medium"
+gobblin.aws.worker.jvm.args="-XX:ReservedCodeCacheSize=100M"
+gobblin.aws.worker.jvm.memory=3G
+# Worker jars location is tied closely to AMI being used. Only change if you know what you are doing
+gobblin.aws.worker.jars.dir="/home/ec2-user/gobblin-lib"
+gobblin.aws.min.workers=2
+gobblin.aws.max.workers=5
+gobblin.aws.desired.workers=2
+
+## Various directories
+gobblin.aws.conf.dir=${gobblin.aws.nfs.root.dir}"/cluster-conf"
+gobblin.aws.master.conf.local=${gobblin.aws.conf.dir}"/log4j-aws.properties,"${gobblin.aws.conf.dir}"/application.conf"
+gobblin.aws.worker.conf.local=${gobblin.aws.master.conf.local}
+# Lib jars location is tied closely to AMI being used. Only change if you know what you are doing
+gobblin.aws.lib.jars.dir="/home/ec2-user/gobblin-lib"
+gobblin.aws.logs.sink.root.dir=${gobblin.aws.nfs.root.dir}"/logs"
+gobblin.aws.work.dir=${gobblin.aws.nfs.root.dir}"/work-dir"
+
+## File system URIs
+fs.uri="file:///"
+writer.fs.uri=${fs.uri}
+state.store.fs.uri=${fs.uri}
+
+## Writer related configuration properties
+writer.destination.type=HDFS
+writer.output.format=AVRO
+writer.staging.dir=${gobblin.aws.work.dir}"/task-staging"
+writer.output.dir=${gobblin.aws.work.dir}"/task-output"
+
+## Data publisher related configuration properties
```

Comment:
does data not get published to s3 at the end of the job? if we just add `data.publisher.fs.uri=s3a://gobblin-bucket-name/` it should work